### PR TITLE
layers: Remove Warning from Enum String helper

### PR DESCRIPTION
**Pull request by spencer-lunarg**
Try and remove the `#pragma warning( disable : 4065 )` warning from `vk_enum_string_helper.h` as not sure if needed anymore

### CODE REVIEW

**Comment by spencer-lunarg** [comment]
seems things like `VkAcquireProfilingLockFlagBitsKHR` are empty (because they were added for the future)

Will update the scripts to just not create a `string_` for these (and still try and remove the warning)

File: scripts/generators/enum_string_helper_generator.py [hunk 1]
```diff
@@ -79,7 +75,8 @@ def generate(self):
         self.write("".join(out))
 
         out = []
-        for enum in self.vk.enums.values():
+        # If there are no fields (empty enum) ignore
+        for enum in filter(lambda x: len(x.fields) > 0, self.vk.enums.values()):
```
Comment (arno-lunarg) [inline scripts/generators/enum_string_helper_generator.py:79]: Could this use the syntax Bob mentioned?
```python
for enum in [x in self.vk.enums.values() if len(x.fields) > 0]:
```

**Review by arno-lunarg — APPROVED** [review]
LGTM, 1 suggestion

File: scripts/generators/enum_string_helper_generator.py (hunk 1 above)
Comment (spencer-lunarg) [reply scripts/generators/enum_string_helper_generator.py:79]: yes, I need to go back through and replace all the `filter` I added

**Comment by spencer-lunarg** [comment]
seems these warnings were hidding issues in `spirv_validation_helper` too... :cry:

**Review by juan-lunarg — APPROVED** [review]

**Pull request merged**